translations/stripes-smart-components/en.json

```diff
@@ -189,5 +182,34 @@
   "addressEdit.label.zipCode": "Zip/Postal Code",
   "addressEdit.label.country": "Country",
   "addressEdit.label.city": "City",
-  "type": "Type"
+  "type": "Type",
+  "customFields": "Custom fields",
+  "customFields.new": "New",
```

Comment:
`stripes-components.button.new` and `stripes-components.button.edit` can be used for things like this.